PC/winsound.c

```diff
@@ -94,17 +94,25 @@ winsound_PlaySound_impl(PyObject *module, PyObject *sound, int flags)
             return NULL;
         }
         wsound = (wchar_t *)view.buf;
+    } else if (PyBytes_Check(sound)) {
+        PyErr_Format(PyExc_TypeError,
+                     "'sound' must be str, os.PathLike, or None; not '%s'",
```

Comment:
Minor fix:
```suggestion
                     "'sound' must be str, os.PathLike, or None, not '%s'",
```

Bigger question: What if we have a subclass of bytes that also is a PathLike resolving to a str? How is that case handled elsewhere?

Reply:
AFAIK the PathLike in test_winsound.py::PlaySoundTest.test_snd_filepath() (on line 129) resolves to str, so we have that covered

Reply:
That one isn't a bytes subclass.

Reply:
i see. an object fitting that description would pass straight through PyOS_FSPath, without calling __fspath__ on it first. `open()` would exhibit the same behavior, since open() passes a pyobject to PyOS_FSPath as well without checking for this case.

Reply:
see _io_open_impl in _iomodule.c

Reply:
@JelleZijlstra to recap, `open()` would treat that kind of object by looking at the bytes object `x` itself, and would not try `x.__fspath__()`.  this code, as it stands, does the same thing. what should we do?